Development/nmos/ocsp_response_handler.h

```diff
@@ -0,0 +1,29 @@
+#ifndef NMOS_OCSP_RESPONSE_HANDLER_H
+#define NMOS_OCSP_RESPONSE_HANDLER_H
+
+#include <cstdint>
+#include <functional>
+#include <vector>
+
+namespace slog
+{
+    class base_gate;
+}
+
+namespace nmos
+{
+    namespace experimental
+    {
+        struct ocsp_state;
+    }
+
+    // callback to return OCSP response in DER format
+    // this callback is executed for each new TLS connection
+    // this callback should not throw exceptions
+    typedef std::function<std::vector<uint8_t>()> ocsp_response_handler;
+
+    // construct callback to receive OCSP response
```

Comment:
Did you mean retrieve rather than receive?

```suggestion
    // construct callback to retrieve OCSP response
```

(Comment should also go in the cpp for consistency.)